source/scene.js

```diff
@@ -146,7 +146,8 @@ function calcGridBounds( grid ) {
 this.executeBlock = function ( block, action ) {
     var blockName = block[ 0 ];
     var blockID = block[ 1 ];
-    this.blockExecuted( blockName, blockID );
+    //SJF: Was breaking procedures
+    // this.blockExecuted( blockName, blockID );
```

Comment:
If we intend to keep tracing, it will need to work for procedures.

@kadst43 Any change in thinking on the necessity/usefulness of blockly tracing?